subprojects/dependency-management/src/main/java/org/gradle/api/internal/runtimeshaded/PackageListGenerator.java

```diff
@@ -66,7 +66,8 @@
         "org/apache/xerces",
         "org/w3c/dom",
         "org/xml/sax",
-        "sun/misc");
+        "sun/misc"
+    );
```

Comment:
:o: unrelated change could be removed from this PR

Reply:
Thanks.